### PR TITLE
Scancodes for Input and other keyboard input improvements

### PR DESCRIPTION
**Pull request by DanielGibson**
Fixed several things that have to do with `K_*` constants and add `K_SC_*` for scancodes that are used as fallbacks for otherwise unknown keys.
See commit messages for more details.

### CODE REVIEW

File: src/server/sv_main.c [hunk 1]
```diff
@@ -339,6 +339,12 @@ SV_RunGameFrame(void)
 	{
 		time_before_game = Sys_Milliseconds();
 	}
+	/*{
+		static int gametime = 0;
+		int newtime = Sys_Milliseconds();
+		printf("since last gameframe: %d ms\n", newtime - gametime);
+		gametime = newtime;
+	}*/
```
Comment (Yamagi) [inline src/server/sv_main.c:347]: Shouldn't this be removed? Or commented in and hidden behind something like `developer`?

Comment (DanielGibson) [reply src/server/sv_main.c:347]: oh, yes, I didn't mean to commit that

**Comment by DanielGibson** [comment]
Added another commit, see commit message.

I'm sorry.

**Comment by DanielGibson** [comment]
Now also includes a hack to support the "console key" on almost all keyboard layouts and fixes that allow binding keys with keycodes that are "special" in configs: `;`, `$` and `"` (and, just to be sure, `'`)

**Comment by DanielGibson** [comment]
According to https://github.com/yquake2/yquake2/issues/543#issuecomment-612662094 this now works as intended (or even better), @Yamagi feel free to merge unless you have additional objections :)

**Comment by Yamagi** [comment]
This looks fine and should work as intended. If there're some unwanted site effects we can fix them in master.

**Pull request merged**